packages/core/src/browser/common-frontend-contribution.ts

```diff
@@ -528,9 +533,13 @@ export class CommonFrontendContribution implements FrontendApplicationContributi
             order: '0'
         });
         registry.registerMenuAction(CommonMenus.VIEW_LAYOUT, {
-            commandId: CommonCommands.COLLAPSE_ALL_PANELS.id,
+            commandId: CommonCommands.TOGGLE_STATUS_BAR.id,
             order: '1'
         });
+        registry.registerMenuAction(CommonMenus.VIEW_LAYOUT, {
+            commandId: CommonCommands.COLLAPSE_ALL_PANELS.id,
+            order: '2'
```

Comment:
@vince-fugnitto I didn't catch what you mean to change - the command with the id `CommonCommands.COLLAPSE_ALL_PANELS.id` not fit the label 'Toggle Status Bar', I added this label to the id `CommonCommands.TOGGLE_STATUS_BAR.id` as I understood, Please tell me if I'm wrong...

Reply:
Yes, since vscode uses a different label for the command and menu, we need to do the same and define `label` for each:

- command: `Toggle Status Bar Visibility`

and

- menu: `Toggle Status Bar`